packages/botfuel-dialog/src/adapters/web-adapter.js

```diff
@@ -137,6 +147,54 @@ class WebAdapter extends Adapter {
   getBody() {
     throw new MissingImplementationError();
   }
+
+  /**
+   * Get absolute url for resource inside static folder
+   * @async
+   * @param {String} resourcePath - resource path relative to static, ex: images/logo.png
+   * @returns {null}
+   */
+  static getStaticUrl(resourcePath) {
+    logger.debug('getStaticUrl', resourcePath);
+    return url.resolve(STATIC_BASE_URL, resourcePath);
+  }
+
+  /**
+   * Get absolute path for template
+   * @async
+   * @param {String} templateName - handlebars template name
+   * @param {Object} params - parameters to be passed to the template
+   * @returns {null}
+   */
+  static getTemplateUrl(templateName, params) {
+    const templateRoot = url.resolve(TEAMPLATE_BASE_URL, templateName);
+    const templateUrl = `${templateRoot}?${querystring.stringify(params)}`;
+    logger.debug('getTemplateUrl', templateUrl);
+
+    return templateUrl;
+  }
+
+  /**
+   * Get absolute path for the screenshot template image
+   * @async
+   * @param {String} templateName - handlebars template name
+   * @param {Object} params - parameters to be passed to the template
+   * @param {Number} width - image width
+   * @param {Number} height - image height
+   * @param {Number} quality - image quality
+   * @returns {null}
+   */
+  static getTemplateImageUrl(templateName, params, width = 800, height = 600, quality = 100) {
```

Comment:
what about getImageUrl

Reply:
I think that getImageUrl will be a little bit general. In the doc we explain to users that they can create an image using a HTLM template which will be screenshot, so I think getTemplateImageUrl will be clearer, but I agree that this is not the best choice. Michel used getScreenshotUrl in banque casino.

Reply:
What about getImageUrl  and in case 1 parameters like getImageUrl(“picture.png”) ==> return “static/picture.png”, and in case getImageUrl(“product.handlebars”, params, width....) then call the dynamic image. So we expose only one function.